react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable prefer-const */
+import React, { useState } from 'react';
+
+export default function AppDrawer() {
+  const [iconClicked, setIconClicked] = useState(false);
```

Comment:
Would `isOpen` be a better name? It can be clicked and still be closed, for example.